### ISSUE TITLE
Failed to activate the formatter package

### ISSUE THREAD

**Issue by tmray** [AtomFormatter/formatter#7]
[Enter steps to reproduce below:]
1. ...
2. ...

**Atom Version**: 1.11.0
**Electron Version**: 0.37.8
**System**: Mac OS X 10.11.6
**Thrown From**: [formatter](https://github.com/atom-community/formatter) package, v2.12.3
### Stack Trace

Failed to activate the formatter package

```
At Arguments to CompositeDisposable.add must have a .dispose() method

TypeError: Arguments to CompositeDisposable.add must have a .dispose() method
    at assertDisposable (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/event-kit/lib/composite-disposable.js:70:13)
    at CompositeDisposable.module.exports.CompositeDisposable.add (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/event-kit/lib/composite-disposable.js:43:11)
    at Module.consumeFormatter (/Users/tomray/.atom/packages/formatter/lib/main.coffee:70:21)
    at Provider.module.exports.Provider.provide (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/service-hub/lib/provider.js:29:52)
    at ServiceHub.module.exports.ServiceHub.consume (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/service-hub/lib/service-hub.js:50:18)
    at Package.module.exports.Package.activateServices (/Applications/Atom.app/Contents/Resources/app.asar/src/package.js:363:75)
    at Package.module.exports.Package.activateNow (/Applications/Atom.app/Contents/Resources/app.asar/src/package.js:186:16)
    at /Applications/Atom.app/Contents/Resources/app.asar/src/package.js:156:32
    at Package.module.exports.Package.measure (/Applications/Atom.app/Contents/Resources/app.asar/src/package.js:92:15)
    at /Applications/Atom.app/Contents/Resources/app.asar/src/package.js:149:26
    at Package.module.exports.Package.activate (/Applications/Atom.app/Contents/Resources/app.asar/src/package.js:146:34)
    at PackageManager.module.exports.PackageManager.activatePackage (/Applications/Atom.app/Contents/Resources/app.asar/src/package-manager.js:538:21)
    at /Applications/Atom.app/Contents/Resources/app.asar/src/package-manager.js:519:29
    at Config.module.exports.Config.transactAsync (/Applications/Atom.app/Contents/Resources/app.asar/src/config.js:337:18)
    at PackageManager.module.exports.PackageManager.activatePackages (/Applications/Atom.app/Contents/Resources/app.asar/src/package-manager.js:514:19)
    at PackageManager.module.exports.PackageManager.activate (/Applications/Atom.app/Contents/Resources/app.asar/src/package-manager.js:497:46)
    at /Applications/Atom.app/Contents/Resources/app.asar/src/atom-environment.js:779:28
```
### Commands

```
```
### Config

``` json
{
  "core": {
    "disabledPackages": [
      "linter-tidy"
    ],
    "projectHome": "/Users/tomray/Projects",
    "themes": [
      "atom-light-ui",
      "atom-light-syntax"
    ]
  }
}
```
### Installed Packages

``` coffee
# User
atom-formatter-jsbeautify, v1.1.2 (inactive)
emmet, v2.4.3 (inactive)
formatter, v2.12.3 (inactive)
linter, v1.11.18 (inactive)
minimap, v4.25.0 (inactive)
pigments, v0.37.0 (inactive)
atom-dark-syntax, v0.27.0 (inactive)
atom-dark-ui, v0.52.0 (inactive)
atom-light-syntax, v0.28.0 (inactive)
atom-light-ui, v0.44.0 (inactive)
base16-tomorrow-dark-theme, v1.2.0 (inactive)
base16-tomorrow-light-theme, v1.2.0 (inactive)
one-dark-ui, v1.6.0 (inactive)
one-light-ui, v1.6.0 (inactive)
one-dark-syntax, v1.3.0 (inactive)
one-light-syntax, v1.3.0 (inactive)
solarized-dark-syntax, v1.0.2 (inactive)
solarized-light-syntax, v1.0.2 (inactive)
about, v1.7.0 (inactive)
archive-view, v0.61.1 (inactive)
autocomplete-atom-api, v0.10.0 (inactive)
autocomplete-css, v0.11.2 (inactive)
autocomplete-html, v0.7.2 (inactive)
autocomplete-plus, v2.31.1 (inactive)
autocomplete-snippets, v1.11.0 (inactive)
autoflow, v0.27.0 (inactive)
autosave, v0.23.1 (inactive)
background-tips, v0.26.1 (inactive)
bookmarks, v0.42.0 (inactive)
bracket-matcher, v0.82.1 (inactive)
command-palette, v0.38.0 (inactive)
deprecation-cop, v0.54.1 (inactive)
dev-live-reload, v0.47.0 (inactive)
encoding-selector, v0.22.0 (inactive)
exception-reporting, v0.40.0 (inactive)
find-and-replace, v0.201.1 (inactive)
fuzzy-finder, v1.4.0 (inactive)
git-diff, v1.1.0 (inactive)
go-to-line, v0.31.0 (inactive)
grammar-selector, v0.48.2 (inactive)
image-view, v0.59.0 (inactive)
incompatible-packages, v0.26.1 (inactive)
keybinding-resolver, v0.35.0 (inactive)
line-ending-selector, v0.5.0 (inactive)
link, v0.31.1 (inactive)
markdown-preview, v0.158.0 (inactive)
metrics, v1.0.0 (inactive)
notifications, v0.65.1 (inactive)
open-on-github, v1.2.0 (inactive)
package-generator, v1.0.0 (inactive)
settings-view, v0.242.2 (inactive)
snippets, v1.0.2 (inactive)
spell-check, v0.68.2 (inactive)
status-bar, v1.4.1 (inactive)
styleguide, v0.47.0 (inactive)
symbols-view, v0.113.1 (inactive)
tabs, v0.101.0 (inactive)
timecop, v0.33.2 (inactive)
tree-view, v0.209.3 (inactive)
update-package-dependencies, v0.10.0 (inactive)
welcome, v0.35.1 (inactive)
whitespace, v0.33.0 (inactive)
wrap-guide, v0.38.2 (inactive)
language-c, v0.52.1 (inactive)
language-clojure, v0.21.0 (inactive)
language-coffee-script, v0.47.2 (inactive)
language-csharp, v0.12.1 (inactive)
language-css, v0.37.1 (inactive)
language-gfm, v0.88.0 (inactive)
language-git, v0.15.0 (inactive)
language-go, v0.42.1 (inactive)
language-html, v0.45.1 (inactive)
language-hyperlink, v0.16.0 (inactive)
language-java, v0.23.0 (inactive)
language-javascript, v0.119.0 (inactive)
language-json, v0.18.2 (inactive)
language-less, v0.29.5 (inactive)
language-make, v0.22.2 (inactive)
language-mustache, v0.13.0 (inactive)
language-objective-c, v0.15.1 (inactive)
language-perl, v0.35.0 (inactive)
language-php, v0.37.2 (inactive)
language-property-list, v0.8.0 (inactive)
language-python, v0.45.0 (inactive)
language-ruby, v0.69.0 (inactive)
language-ruby-on-rails, v0.25.0 (inactive)
language-sass, v0.56.0 (inactive)
language-shellscript, v0.22.4 (inactive)
language-source, v0.9.0 (inactive)
language-sql, v0.23.0 (inactive)
language-text, v0.7.1 (inactive)
language-todo, v0.28.0 (inactive)
language-toml, v0.18.0 (inactive)
language-xml, v0.34.9 (inactive)
language-yaml, v0.26.0 (inactive)

# Dev
No dev packages
```

**Comment by mmssantos** [comment]
Same issue on my end.

**Comment by steinhjertas** [comment]
+1

**Comment by dancinglone** [comment]
+1

**Comment by neuronetpl** [comment]
same on windows 1.11.1

**Comment by ghost** [comment]
Getting same on my Mac 1.11.1

Formatter:  2.12.3

**Comment by darrenfurr** [comment]
Samesies. formatter 2.12.3 & Atom 1.11.2

**Comment by bxb100** [comment]
+1

**Comment by ryanplusplus** [comment]
Same: OS X, Atom 1.11.2, formatter 2.12.3

**Comment by tmray** [comment]
I got tired of seeing this every time I opened it, so I did a complete uninstall of Atom. 
Found how to do that here - [How to Completely Uninstall Atom for Mac?](https://discuss.atom.io/t/how-to-completely-uninstall-atom-for-mac/9084) 

Which was a crazy process BTW. 
"Why sooo many things to remove?" 😱 

Then installed the latest version again and when I opened it the error was gone.

I would prefer not to re-install things to fix a problem. Hopefully, a solution will be found so it doesn't really come down to this in the future.

**Comment by dayfuaim** [comment]
The same. Atom 1.11.2, Ubuntu 16.10, formatter 2.12.3

**Comment by njcaballero** [comment]
+1

**Comment by funny123** [comment]
The same. Atom 1.11.2, Ubuntu 16.10, formatter 2.12.3

**Comment by SecureCloud-biz** [comment]
+1

**Comment by emveeoh** [comment]
Same error.

```
Failed to activate the formatter package
Arguments to CompositeDisposable.add must have a .dispose() method
```

**MY CONFIG:**
Formatter package: 2.12.3
Atom : 1.12.5
Electron: 1.3.9
Chrome : 52.0.2743.82
Node : 6.5.0
apm 1.12.9
npm 3.10.5
node 4.4.5
python 2.7.12
git 2.7.4
Linux Mint 18 Cinnamon (4.4.0-47-generic) (Ubuntu 16.04 LTS)

**Comment by ryanplusplus** [comment]
Can you add a note when this has been released to apm?

**Comment by pmer** [comment]
I was able to fix the issue temporarily by manually applying the changes to my local package. It's only a 5-line diff.

**Comment by oychao** [comment]
Any progress now?

**Comment by abdelouahabb** [comment]
should be reopened !

`[Enter steps to reproduce below:]

1. ...
2. ...

**Atom Version**: 1.13.1
**Electron Version**: 1.3.13
**System**: Unknown Windows Version
**Thrown From**: [formatter](https://github.com/atom-community/formatter) package, v2.12.3


### Stack Trace

Failed to activate the formatter package

```
At Arguments to CompositeDisposable.add must have a .dispose() method

TypeError: Arguments to CompositeDisposable.add must have a .dispose() method
    at assertDisposable (C:\Users\Abdelouahab\AppData\Local\atom\app-1.13.1\resources\app.asar\node_modules\event-kit\lib\composite-disposable.js:70:13)
    at CompositeDisposable.module.exports.CompositeDisposable.add (C:\Users\Abdelouahab\AppData\Local\atom\app-1.13.1\resources\app.asar\node_modules\event-kit\lib\composite-disposable.js:43:11)
    at Module.consumeFormatter (file:///C:/Users/Abdelouahab/.atom/packages/formatter/lib/main.coffee:70:21)
    at Provider.module.exports.Provider.provide (C:\Users\Abdelouahab\AppData\Local\atom\app-1.13.1\resources\app.asar\node_modules\service-hub\lib\provider.js:29:52)
    at ServiceHub.module.exports.ServiceHub.consume (C:\Users\Abdelouahab\AppData\Local\atom\app-1.13.1\resources\app.asar\node_modules\service-hub\lib\service-hub.js:50:18)
    at Package.module.exports.Package.activateServices (C:\Users\Abdelouahab\AppData\Local\atom\app-1.13.1\resources\app.asar\src\package.js:377:75)
    at Package.module.exports.Package.activateNow (C:\Users\Abdelouahab\AppData\Local\atom\app-1.13.1\resources\app.asar\src\package.js:194:16)
    at C:\Users\Abdelouahab\AppData\Local\atom\app-1.13.1\resources\app.asar\src\package.js:164:32
    at Package.module.exports.Package.measure (C:\Users\Abdelouahab\AppData\Local\atom\app-1.13.1\resources\app.asar\src\package.js:94:15)
    at C:\Users\Abdelouahab\AppData\Local\atom\app-1.13.1\resources\app.asar\src\package.js:157:26
    at Package.module.exports.Package.activate (C:\Users\Abdelouahab\AppData\Local\atom\app-1.13.1\resources\app.asar\src\package.js:154:34)
    at PackageManager.module.exports.PackageManager.activatePackage (C:\Users\Abdelouahab\AppData\Local\atom\app-1.13.1\resources\app.asar\src\package-manager.js:550:34)
    at C:\Users\Abdelouahab\AppData\Local\atom\app-1.13.1\resources\app.asar\src\package-manager.js:531:29
    at Config.module.exports.Config.transactAsync (C:\Users\Abdelouahab\AppData\Local\atom\app-1.13.1\resources\app.asar\src\config.js:337:18)
    at PackageManager.module.exports.PackageManager.activatePackages (C:\Users\Abdelouahab\AppData\Local\atom\app-1.13.1\resources\app.asar\src\package-manager.js:526:19)
    at PackageManager.module.exports.PackageManager.activate (C:\Users\Abdelouahab\AppData\Local\atom\app-1.13.1\resources\app.asar\src\package-manager.js:508:46)
    at C:\Users\Abdelouahab\AppData\Local\atom\app-1.13.1\resources\app.asar\src\atom-environment.js:831:28
```

### Commands

```
```

### Config

```json
{
  "core": {
    "autoHideMenuBar": true,
    "customFileTypes": {
      "source.ini": [
        ".buckconfig",
        ".flowconfig",
        ".hgrc"
      ],
      "source.json": [
        ".arcconfig",
        "BUCK.autodeps"
      ],
      "source.python": [
        "BUCK"
      ]
    },
    "disabledPackages": [
      "nuclide-diff-view",
      "nuclide-file-watcher",
      "nuclide-flow",
      "line-ending-converter",
      "tree-view"
    ],
    "telemetryConsent": "no",
    "themes": [
      "seti-ui",
      "atom-monokai"
    ]
  }
}
```

### Installed Packages

```coffee
# User
amp-snippets, v0.1.1 (inactive)
angularjs, v0.3.5 (inactive)
atom-beautify, vundefined (inactive)
atom-formatter-jsbeautify, v1.1.2 (inactive)
atom-html-preview, v0.1.22 (inactive)
atom-monokai, v0.10.7 (inactive)
atom-ternjs, v0.17.0 (inactive)
autocomplete-paths, v1.0.5 (inactive)
autocomplete-python, v1.8.36 (inactive)
color-picker, v2.2.5 (inactive)
emmet, v2.4.3 (inactive)
formatter, v2.12.3 (inactive)
git-time-machine, v1.5.4 (inactive)
highlight-selected, v0.12.0 (inactive)
indent-guide-improved, v1.4.12 (inactive)
jshint, v1.8.6 (inactive)
language-javascript-jsx, v0.3.7 (inactive)
linter, v1.11.21 (inactive)
linter-flake8, v2.1.4 (inactive)
linter-json-lint, v0.1.1 (inactive)
linter-pep8, v1.3.2 (inactive)
linter-pycodestyle, v2.0.0 (inactive)
minimap, v4.26.8 (inactive)
open-in-browsers, v0.0.19 (inactive)
pretty-json, v1.6.1 (inactive)
react, v0.16.2 (inactive)
react-snippets, v0.6.2 (inactive)
script, v3.14.1 (inactive)
seti-ui, v1.4.0 (inactive)
tool-bar, v1.0.1 (inactive)
tool-bar-atom, v1.7.0 (inactive)
tool-bar-config, v0.1.0 (inactive)
tornado-templates, v0.2.0 (inactive)
atom-dark-syntax, v0.28.0 (inactive)
atom-dark-ui, v0.53.0 (inactive)
atom-light-syntax, v0.29.0 (inactive)
atom-light-ui, v0.46.0 (inactive)
base16-tomorrow-dark-theme, v1.4.0 (inactive)
base16-tomorrow-light-theme, v1.4.0 (inactive)
one-dark-ui, v1.8.2 (inactive)
one-light-ui, v1.8.2 (inactive)
one-dark-syntax, v1.6.0 (inactive)
one-light-syntax, v1.6.0 (inactive)
solarized-dark-syntax, v1.1.1 (inactive)
solarized-light-syntax, v1.1.1 (inactive)
about, v1.7.2 (inactive)
archive-view, v0.62.0 (inactive)
autocomplete-atom-api, v0.10.0 (inactive)
autocomplete-css, v0.14.1 (inactive)
autocomplete-html, v0.7.2 (inactive)
autocomplete-plus, v2.33.1 (inactive)
autocomplete-snippets, v1.11.0 (inactive)
autoflow, v0.29.0 (inactive)
autosave, v0.23.2 (inactive)
background-tips, v0.26.1 (inactive)
bookmarks, v0.43.2 (inactive)
bracket-matcher, v0.85.1 (inactive)
command-palette, v0.39.1 (inactive)
deprecation-cop, v0.55.1 (inactive)
dev-live-reload, v0.47.0 (inactive)
encoding-selector, v0.22.0 (inactive)
exception-reporting, v0.40.0 (inactive)
find-and-replace, v0.204.5 (inactive)
fuzzy-finder, v1.4.0 (inactive)
git-diff, v1.2.0 (inactive)
go-to-line, v0.31.2 (inactive)
grammar-selector, v0.48.2 (inactive)
image-view, v0.60.0 (inactive)
incompatible-packages, v0.26.1 (inactive)
keybinding-resolver, v0.35.0 (inactive)
line-ending-selector, v0.5.1 (inactive)
link, v0.31.2 (inactive)
markdown-preview, v0.159.3 (inactive)
metrics, v1.1.3 (inactive)
notifications, v0.65.1 (inactive)
open-on-github, v1.2.1 (inactive)
package-generator, v1.0.2 (inactive)
settings-view, v0.244.0 (inactive)
snippets, v1.0.4 (inactive)
spell-check, v0.68.5 (inactive)
status-bar, v1.6.0 (inactive)
styleguide, v0.48.0 (inactive)
symbols-view, v0.113.1 (inactive)
tabs, v0.103.1 (inactive)
timecop, v0.33.2 (inactive)
tree-view, v0.211.1 (inactive)
update-package-dependencies, v0.10.0 (inactive)
welcome, v0.35.1 (inactive)
whitespace, v0.35.0 (inactive)
wrap-guide, v0.39.0 (inactive)
language-c, v0.54.0 (inactive)
language-clojure, v0.22.1 (inactive)
language-coffee-script, v0.48.1 (inactive)
language-csharp, v0.13.0 (inactive)
language-css, v0.40.1 (inactive)
language-gfm, v0.88.0 (inactive)
language-git, v0.15.0 (inactive)
language-go, v0.43.0 (inactive)
language-html, v0.47.1 (inactive)
language-hyperlink, v0.16.1 (inactive)
language-java, v0.24.0 (inactive)
language-javascript, v0.122.0 (inactive)
language-json, v0.18.3 (inactive)
language-less, v0.29.6 (inactive)
language-make, v0.22.2 (inactive)
language-mustache, v0.13.0 (inactive)
language-objective-c, v0.15.1 (inactive)
language-perl, v0.37.0 (inactive)
language-php, v0.37.3 (inactive)
language-property-list, v0.8.0 (inactive)
language-python, v0.45.1 (inactive)
language-ruby, v0.70.2 (inactive)
language-ruby-on-rails, v0.25.1 (inactive)
language-sass, v0.57.0 (inactive)
language-shellscript, v0.23.0 (inactive)
language-source, v0.9.0 (inactive)
language-sql, v0.25.0 (inactive)
language-text, v0.7.1 (inactive)
language-todo, v0.29.1 (inactive)
language-toml, v0.18.1 (inactive)
language-xml, v0.34.12 (inactive)
language-yaml, v0.27.1 (inactive)

# Dev
No dev packages
```

`

**Comment by njcaballero** [comment]
^^^ I agree should be reopened. still getting this error, I have the latest version.   It shouldn't be closed until the fix is actually released.

**Comment by marmistrz** [comment]
This may be a caused by a buggy provider

**Comment by marmistrz** [comment]
This is still a problem, I can't create a formatter due to this bug.

**Comment by kalinon** [comment]
[Enter steps to reproduce below:]

1. ...
2. ...

**Atom Version**: 1.13.1
**Electron Version**: 1.3.13
**System**: Mac OS X 10.12.3
**Thrown From**: [formatter](https://github.com/atom-community/formatter) package, v2.12.3


### Stack Trace

Failed to activate the formatter package

```
At Arguments to CompositeDisposable.add must have a .dispose() method

TypeError: Arguments to CompositeDisposable.add must have a .dispose() method
    at assertDisposable (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/event-kit/lib/composite-disposable.js:70:13)
    at CompositeDisposable.module.exports.CompositeDisposable.add (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/event-kit/lib/composite-disposable.js:43:11)
    at Module.consumeFormatter (/Users/homans/.atom/packages/formatter/lib/main.coffee:70:21)
    at Provider.module.exports.Provider.provide (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/service-hub/lib/provider.js:29:52)
    at ServiceHub.module.exports.ServiceHub.consume (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/service-hub/lib/service-hub.js:50:18)
    at Package.module.exports.Package.activateServices (/Applications/Atom.app/Contents/Resources/app.asar/src/package.js:377:75)
    at Package.module.exports.Package.activateNow (/Applications/Atom.app/Contents/Resources/app.asar/src/package.js:194:16)
    at /Applications/Atom.app/Contents/Resources/app.asar/src/package.js:164:32
    at Package.module.exports.Package.measure (/Applications/Atom.app/Contents/Resources/app.asar/src/package.js:94:15)
    at /Applications/Atom.app/Contents/Resources/app.asar/src/package.js:157:26
    at Package.module.exports.Package.activate (/Applications/Atom.app/Contents/Resources/app.asar/src/package.js:154:34)
    at PackageManager.module.exports.PackageManager.activatePackage (/Applications/Atom.app/Contents/Resources/app.asar/src/package-manager.js:550:34)
    at /Applications/Atom.app/Contents/Resources/app.asar/src/package-manager.js:531:29
    at Config.module.exports.Config.transactAsync (/Applications/Atom.app/Contents/Resources/app.asar/src/config.js:337:18)
    at PackageManager.module.exports.PackageManager.activatePackages (/Applications/Atom.app/Contents/Resources/app.asar/src/package-manager.js:526:19)
    at PackageManager.module.exports.PackageManager.activate (/Applications/Atom.app/Contents/Resources/app.asar/src/package-manager.js:508:46)
    at /Applications/Atom.app/Contents/Resources/app.asar/src/atom-environment.js:831:28
```

### Commands

```
     -0:00.3.0 tree-view:show (atom-workspace.workspace.scrollbars-visible-always.theme-one-dark-syntax.theme-one-dark-ui)
```

### Config

```json
{
  "core": {
    "disabledPackages": [
      "go-debug"
    ],
    "ignoredNames": [
      ".git",
      ".hg",
      ".svn",
      ".DS_Store",
      "._*",
      "Thumbs.db",
      ".pioenvs",
      ".piolibdeps",
      ".clang_complete",
      ".gcc-flags.json"
    ],
    "projectHome": "C:\\Users\\homans\\work",
    "telemetryConsent": "no"
  }
}
```

### Installed Packages

```coffee
# User
atom-formatter-jsbeautify, v1.1.2 (inactive)
autocomplete-clang, v0.10.0 (inactive)
autocomplete-go, v1.3.0 (inactive)
build, v0.67.0 (inactive)
builder-go, v1.0.1 (inactive)
busy, v0.7.0 (inactive)
docker, v0.7.0 (inactive)
file-icons, v2.0.14 (inactive)
formatter, v2.12.3 (inactive)
formatter-json, v1.0.1 (inactive)
go-config, v1.2.4 (inactive)
go-debug, v1.0.4 (inactive)
go-get, v2.1.1 (inactive)
go-plus, v5.1.1 (inactive)
go-rename, v0.3.0 (inactive)
godoc, v1.1.1 (inactive)
gofmt, v1.2.0 (inactive)
gometalinter-linter, v1.2.2 (inactive)
gorename, v1.1.0 (inactive)
html2haml, v0.10.0 (inactive)
language-docker, v1.1.7 (inactive)
language-generic-config, v1.3.0 (inactive)
language-haml, v0.24.1 (inactive)
language-ini, v1.17.0 (inactive)
language-nginx, v0.6.2 (inactive)
language-protobuf, v0.7.1 (inactive)
linter, v1.11.21 (inactive)
linter-docker, v0.1.4 (inactive)
linter-gcc, v0.7.1 (inactive)
navigator-go, v1.1.3 (inactive)
perltidy, v3.0.0 (inactive)
tester-go, v2.0.2 (inactive)
tokamak-terminal, v10.0.1 (inactive)
tool-bar, v1.0.1 (inactive)
atom-dark-syntax, v0.28.0 (inactive)
atom-dark-ui, v0.53.0 (inactive)
atom-light-syntax, v0.29.0 (inactive)
atom-light-ui, v0.46.0 (inactive)
base16-tomorrow-dark-theme, v1.4.0 (inactive)
base16-tomorrow-light-theme, v1.4.0 (inactive)
one-dark-ui, v1.8.2 (inactive)
one-light-ui, v1.8.2 (inactive)
one-dark-syntax, v1.6.0 (inactive)
one-light-syntax, v1.6.0 (inactive)
solarized-dark-syntax, v1.1.1 (inactive)
solarized-light-syntax, v1.1.1 (inactive)
about, v1.7.2 (inactive)
archive-view, v0.62.0 (inactive)
autocomplete-atom-api, v0.10.0 (inactive)
autocomplete-css, v0.14.1 (inactive)
autocomplete-html, v0.7.2 (inactive)
autocomplete-plus, v2.33.1 (inactive)
autocomplete-snippets, v1.11.0 (inactive)
autoflow, v0.29.0 (inactive)
autosave, v0.23.2 (inactive)
background-tips, v0.26.1 (inactive)
bookmarks, v0.43.2 (inactive)
bracket-matcher, v0.85.1 (inactive)
command-palette, v0.39.1 (inactive)
deprecation-cop, v0.55.1 (inactive)
dev-live-reload, v0.47.0 (inactive)
encoding-selector, v0.22.0 (inactive)
exception-reporting, v0.40.0 (inactive)
find-and-replace, v0.204.5 (inactive)
fuzzy-finder, v1.4.0 (inactive)
git-diff, v1.2.0 (inactive)
go-to-line, v0.31.2 (inactive)
grammar-selector, v0.48.2 (inactive)
image-view, v0.60.0 (inactive)
incompatible-packages, v0.26.1 (inactive)
keybinding-resolver, v0.35.0 (inactive)
line-ending-selector, v0.5.1 (inactive)
link, v0.31.2 (inactive)
markdown-preview, v0.159.3 (inactive)
metrics, v1.1.3 (inactive)
notifications, v0.65.1 (inactive)
open-on-github, v1.2.1 (inactive)
package-generator, v1.0.2 (inactive)
settings-view, v0.244.0 (inactive)
snippets, v1.0.4 (inactive)
spell-check, v0.68.5 (inactive)
status-bar, v1.6.0 (inactive)
styleguide, v0.48.0 (inactive)
symbols-view, v0.113.1 (inactive)
tabs, v0.103.1 (inactive)
timecop, v0.33.2 (inactive)
tree-view, v0.211.1 (inactive)
update-package-dependencies, v0.10.0 (inactive)
welcome, v0.35.1 (inactive)
whitespace, v0.35.0 (inactive)
wrap-guide, v0.39.0 (inactive)
language-c, v0.54.0 (inactive)
language-clojure, v0.22.1 (inactive)
language-coffee-script, v0.48.1 (inactive)
language-csharp, v0.13.0 (inactive)
language-css, v0.40.1 (inactive)
language-gfm, v0.88.0 (inactive)
language-git, v0.15.0 (inactive)
language-go, v0.43.0 (inactive)
language-html, v0.47.1 (inactive)
language-hyperlink, v0.16.1 (inactive)
language-java, v0.24.0 (inactive)
language-javascript, v0.122.0 (inactive)
language-json, v0.18.3 (inactive)
language-less, v0.29.6 (inactive)
language-make, v0.22.2 (inactive)
language-mustache, v0.13.0 (inactive)
language-objective-c, v0.15.1 (inactive)
language-perl, v0.37.0 (inactive)
language-php, v0.37.3 (inactive)
language-property-list, v0.8.0 (inactive)
language-python, v0.45.1 (inactive)
language-ruby, v0.70.2 (inactive)
language-ruby-on-rails, v0.25.1 (inactive)
language-sass, v0.57.0 (inactive)
language-shellscript, v0.23.0 (inactive)
language-source, v0.9.0 (inactive)
language-sql, v0.25.0 (inactive)
language-text, v0.7.1 (inactive)
language-todo, v0.29.1 (inactive)
language-toml, v0.18.1 (inactive)
language-xml, v0.34.12 (inactive)
language-yaml, v0.27.1 (inactive)

# Dev
No dev packages
```

**Comment by nfourtythree** [comment]
For anyone still getting this issue the [fix has been merged into the repo](https://github.com/atom-community/formatter/pull/8/commits/d136a4a8aeebc1735bd80080d44926b4d312a8ef) but looks there hasn't been a new release that includes the fix.

If you make the changes in the fix manually following the steps below the error should stop.

- Open settings
- Click `Open Config Folder`
- Go to `Packages > formatter`
- Make the changes in the two files
- Reload atom

**Comment by marmistrz** [comment]
@nfourtythree You meant *a new release has **not** been made*?

**Comment by nfourtythree** [comment]
@marmistrz that is exactly what I meant, good spot! Will update my comment now

**Comment by marmistrz** [comment]
@nfourtythree Who is responsible for making releases?

**Comment by nfourtythree** [comment]
@marmistrz it looks like @basarat has been responsible for the releases

**Comment by marmistrz** [comment]
@nfourtythree he'll get notified about this mention here, won't he? I think it's important to make an official release unless we want another twenty people reporting that this doesn't work.

**Comment by winstliu** [comment]
Published as v2.12.4.  Sorry for the delay.
